stream-chat-android-ui-components/api/stream-chat-android-ui-components.api

```diff
@@ -882,7 +882,6 @@ public final class io/getstream/chat/android/ui/message/list/MessageListItemStyl
 	public final fun getStyleTextColor (Z)Ljava/lang/Integer;
 	public final fun getThreadsEnabled ()Z
 	public fun hashCode ()I
-	public final fun setReactionsEnabled (Z)V
```

Comment:
These changes were made in https://github.com/GetStream/stream-chat-android/pull/1602, this file just wasn't updated there